public/video-ui/src/components/FormFields/FormFieldBylinePicker.js

```diff
@@ -0,0 +1,261 @@
+import React from 'react';
+import ContentApi from '../../services/capi';
+import { tagsFromStringList, tagsToStringList } from '../../util/tagParsers';
+import removeStringTagDuplicates from '../../util/removeStringTagDuplicates';
+
+export default class FormFieldBylinePicker extends React.Component {
```

Comment:
I was going to change the whole name in the pr that makes this component a more generic tag picker so maybe it is enough to just do it then?